Cargo.toml

```diff
@@ -12,9 +12,16 @@ exclude = [ ".gitignore", "TESTVECTORS" ]
 
 [dependencies]
 arrayref = "0.3.2"
-rust-crypto = "^0.2"
-rand = "^0.3"
-curve25519-dalek = "^0.1"
+sha2 = "^0.1"
+curve25519-dalek = { git = "https://github.com/chain/curve25519-dalek", branch = "no-std" }
```

Comment:
Will need to merge this first:

https://github.com/isislovecruft/curve25519-dalek/pull/17